examples/opt_serving/textgen_demo.py

```diff
@@ -1,27 +1,35 @@
 """Use huggingface/transformers interface and Alpa backend for distributed inference."""
 from transformers import AutoTokenizer
 from opt_serving.model.wrapper import get_model
+import numpy as np
 
 # Load the tokenizer. We have to use the 30B version because
 # other versions have some issues. The 30B version works for all OPT models.
 tokenizer = AutoTokenizer.from_pretrained("facebook/opt-30b", use_fast=False)
 tokenizer.add_bos_token = False
 
-generate_params = {"do_sample": True, "num_beams": 4, "num_return_sequences": 3}
+generate_params = {"do_sample": False, "num_beams": 1, "num_return_sequences": 1}
```

Comment:
If batch compatible with beam search?
If not, where should we raise errors?

Reply:
It should be compatible with beam search.

Reply:
it works fine.